docs/source/gbd2019_models/risk_attributable_causes/protein_energy_malnutrition/index.rst

```diff
@@ -4,6 +4,72 @@
 Protein Energy Malnutrition
 ===========================
 
+Protein-energy malnutrition (PEM) is an acute form of undernutrition. 
+Undernutrition can take a variety of forms, including stunting (low height for 
+age), wasting (low weight for height), underweight (low weight for age), and 
+micronutrient deficiencies (a lack of key vitamins or minerals). PEM refers 
+specifically to an acute (as opposed to chronic) disorder that results in 
+wasting, and is classified as either marasmus, kwashiorkor, or some 
+combination of these. [UpToDate-malnutrition]_ [WHO-Malnutrition]_ 
+[emedicine-PEM]_
+
```

Comment:
We should emphasize the difference in clinical diagnosis and the anthropometric diagnosis. So we can add this clarification: 

```suggestion

Acute malnutrition pertains to a group of linked disorders that includes kwashiorkor, marasmus, and intermediate states of marasmic kwashiorkor. They are distinguished based on **clinical** findings, with the primary distinction between kwashiorkor and marasmus being the presence of edema in kwashiorkor. Note that acute malnutrition should be diagnosed using anthropometrics. [reference Dipasquale Nutrients 2020]

**Moderate acute malnutrition**: WHZ < -2 but > -3 AND no oedema
**Severe acute malnutrition**: WHZ < -3  OR bilateral pitting oedema (kwashiorkor)

**Note**: Although GBD categorizes those with `WHZ scores < -2 but > -3 with oedema` as MAM, these individuals are treated as severe. 

[reference WHO: https://www.who.int/elena/titles/full_recommendations/sam_management/en/] and 
WHO pocket book: https://www-ncbi-nlm-nih-gov.offcampus.lib.washington.edu/books/NBK154454/]

```